airflow/jobs/scheduler_job.py

```diff
@@ -66,7 +67,7 @@ class DagFileProcessor(AbstractDagFileProcessor, LoggingMixin):
     :param dag_id_white_list: If specified, only look at these DAG ID's
     :type dag_id_white_list: list[unicode]
     :param zombies: zombie task instances to kill
-    :type zombies: list[airflow.utils.dag_processing.SimpleTaskInstance]
+    :type zombies: list[airflow.executors.executor_types.SimpleTaskInstance]
```

Comment:
Why don't you use `SimpleTaskInstance` directly here?

Reply:
(since you imported it already)

Reply:
Types for doc comments don't respect/use imports - they create links based on the package name only

(i.e. mypy respects imports, sphinx doesn't and needs full package names to create a link when rendering.)

Reply:
Since we are adding more and more typehints I am going to addd sphinx-autodoc-typehints extensions and get rid of the docstring types in the affected files.